CMakeLists.txt

```diff
@@ -7,9 +7,14 @@ CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
 
 PROJECT(OpenTomb)
 
-SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -msse3")
+IF( ${CMAKE_COMPILER_IS_GNUCC} )
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -msse3")
+ENDIF()
+IF( ${CMAKE_CXX_COMPILER_ID} STREQUAL "Clang")
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wextra -msse3")
+ENDIF()
 
-add_subdirectory(3rdparty)
+# add_subdirectory(3rdparty) there is no CMakeLists.txt in 3rdparty
```

Comment:
OK, let's blame @Lwmte for not taking a proper look at the old CMakeLists.txt in d36177d28f693fb53440af5e6e3349f5cac8984a :)